requirements/requirements-common.txt

```diff
@@ -7,3 +7,6 @@ selenium==2.53.6
 coverage
 psutil
 ruamel.yaml==0.15.23
+channels==1.1.6
+asgi-redis==1.4.2
+Twisted[tls,http2]
```

Comment:
I'd recommend adding a version

Reply:
Cannot add since lots of dependencies are installed. Don't know the specific version of all the dependencies